### ISSUE TITLE
Correct example result for `engine_exchangeTransitionConfigurationV1`

### ISSUE THREAD

**Issue by alexandratran** [hyperledger/besu-docs#1388]
Remove the incorrect `"payloadId": null` at the end of the example JSON result for [`engine_exchangeTransitionConfigurationV1`](https://besu.hyperledger.org/development/public-networks/reference/engine-api#engine_exchangetransitionconfigurationv1).

The example result should just be:
```
{
  "jsonrpc": "2.0",
  "id": 67,
  "result": {
    "terminalTotalDifficulty": 0,
    "terminalBlockHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "terminalBlockNumber": "0x1"
  }
}
```

**Comment by Vikramop** [comment]
Hi, can you guide me to the file where the changes to be made.

**Comment by alexandratran** [comment]
@Vikramop the file is linked in the PR description above: https://besu.hyperledger.org/development/public-networks/reference/engine-api#engine_exchangetransitionconfigurationv1

**Comment by Vikramop** [comment]
Thank you for guiding, Btw I have raised my pr.
https://github.com/hyperledger/besu-docs/pull/1389

**Comment by alexandratran** [comment]
fixed by #1389
